youtube_dl/extractor/vlive.py

```diff
@@ -261,3 +265,62 @@ def _real_extract(self, url):
 
         return self.playlist_result(
             entries, channel_code, channel_name)
+
+
+class VLivePlaylistIE(InfoExtractor):
+    IE_NAME = 'vlive:playlist'
+    _VALID_URL = r'https?://(?:(?:www|m)\.)?vlive\.tv/video/(?P<video_id>[0-9]+)/playlist/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'http://www.vlive.tv/video/30824/playlist/30826',
+        'info_dict': {
+            'id': '30826',
+            'title': 'TWICE TV5 - TWICE in SWITZERLAND'
+        },
+        'playlist_mincount': 20
+    }, {
+        'url': 'http://www.vlive.tv/video/22867/playlist/22912',
```

Comment:
Both test the same extraction scenario. The rule is one test per extraction scenario.